src/js/actions/layers.js

```diff
@@ -1840,20 +1861,17 @@ define(function (require, exports) {
         // and shifts all the layers correctly
         _autoCanvasResizeShiftHandler = function (event) {
             var applicationStore = this.flux.store("application"),
-                currentDocument = applicationStore.getCurrentDocument();
-            
-            if (currentDocument !== null) {
-                var payload = {
-                    documentID: applicationStore.getCurrentDocumentID(),
-                    layerIDs: collection.pluck(currentDocument.layers.all, "id"),
-                    position: {
-                        x: event.to.horizontal,
-                        y: event.to.vertical
-                    }
+                currentDocument = applicationStore.getCurrentDocument(),
+                position = {
+                    x: event.to.horizontal,
+                    y: event.to.vertical
                 };
 
-                this.dispatch(events.document.TRANSLATE_LAYERS, payload);
+            if (!currentDocument) {
```

Comment:
I was trying to fit into our variable model.. but yeah, for event handler actions we can probably be outside the norm. Will make the changes.